csuntechlab/iSTART
# STR-229 -- First Slide of Module resets to the Top

# Description
Screen will go back to the top after visiting the first slide of any Module
  
# Testing
1. Check on modern browsers + Safari.
2. Ensure the page resets to the top when visiting `Dashboard` to any `Module`.
